lib/smart_app_launch/token_response_body_test_stu2_2.rb

```diff
@@ -12,7 +12,14 @@ class TokenResponseBodyTestSTU22 < TokenResponseBodyTest
 
       The format of the optional `fhirContext` field is validated if present.
     )
+
     id :smart_token_response_body_stu2_2
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@73',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@74',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@75',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@78',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@84',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@172'
```

Comment:
only partial coverage of @172 - this checks that it's an array but not that it's not Patient or Encounter.
Needs an attestation for the second half.

**automation attestation combo**

Reply:
Should I remove 172 annotation?